packages/ozone/tests/scheduled-action-processor.test.ts

```diff
@@ -74,7 +81,14 @@ describe('scheduled action processor', () => {
       const testSubject = sc.dids.alice
 
       const pastTime = new Date(Date.now() - 1000).toISOString()
-      await scheduleTestAction(testSubject, { executeAt: pastTime })
+      await scheduleTestAction(
```

Comment:
I think this change would leave the non-email code path untested? May be worth having an explicit test to ensure scheduled actions that don't have email data don't trigger an email send?

Also, I made this comment in https://github.com/bluesky-social/atproto/pull/4221 but adding in mailpit to the dev-infra would allow you to actually test that the email was indeed sent. There's also some code in that PR for asserting sent emails with the current infrastructure in a clean-ish way.